modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java

```diff
@@ -28,6 +30,39 @@
  */
 public class ObjectDefinitionTestUtil {
 
+	public static ObjectDefinition addCustomObjectDefinition()
```

Comment:
@p-albuquerque it's not clear that we are also publishing the object definition.

Reply:
rename it to `publishCustomObjectDefinition`